src/qt/splashscreen.cpp

```diff
@@ -64,7 +64,7 @@ SplashScreen::SplashScreen(interfaces::Node& node, Qt::WindowFlags f, const Netw
     pixPaint.fillRect(rGradient, gradient);
 
     // draw the bitcoin icon, expected size of PNG: 1024x1024
-    QRect rectIcon(QPoint(-150,-122), QSize(430,430));
+    QRect rectIcon(QPoint(-15,-12), QSize(270,270));
```

Comment:
Nit: Please `clang-format -i` to get proper social distancing between `-15` and `-12`.

Same comment also applies to `270` and `270` (self-quarantine).

Reply:
And why isn't this variable called rektIcon?